src/core/database/query/match-project-based-props.ts

```diff
@@ -98,6 +102,13 @@ export const matchProjectSens = (projectVar: string) => (query: Query) =>
       // Prevent single row with project from expanding to more here via multiple engagement matches
       .raw('LIMIT 1')
       .return(coalesce('langSens.value', '"High"').as('sensitivity'))
+      // If the cardinality of this subquery is zero, no rows will be returned at all.
+      // So, if no projects are matched (optional matching), we still need to have a cardinality > 0 in order to continue
+      // https://neo4j.com/developer/kb/conditional-cypher-execution/#_the_subquery_must_return_a_row_for_the_outer_query_to_continue
+      .union()
+      .with(projectVar)
+      .with(projectVar)
+      .return('"High" as sensitivity')
```

Comment:
Why do we need sensitivity without project?

Reply:
I believe you have to import the same stuff into each union-ed query, but I'll double check that.

Reply:
I think we just need to add a where clause in the 3rd union so that it doesn’t return any rows if there is a project. But looks good otherwise!